modules/redis-cluster/main.tf

```diff
@@ -32,6 +32,10 @@ resource "google_redis_cluster" "redis_cluster" {
     }
   }
 
+  zone_distribution_config {
```

Comment:
@lanandra can u plz make it dynamic block and also add `zone` to it?

Reply:
if I don't see any progress by end of week I'm gonna hijack this and create a new PR with suggested changes :D lol